signup.css

```diff
@@ -0,0 +1,226 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;0,500;0,700;0,900;1,400;1,500&display=swap');
+
+body {
+  background-color: rgb(236, 238, 241);
+  font-family: 'Roboto', sans-serif;
+}
+
+main {
+  margin: 0 auto;
+  margin-top: 2px;
+}
+
+header {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.reset-account {
+  margin-right: 20px;
+}
+
+.reset-account p {
+  color: rgb(80, 80, 80);
+  font-size: 12px;
+  margin-bottom: 2px;
+  margin-top: 1px;
+}
+
+.reset-account a {
+  text-decoration: none;
+  color: rgb(0, 119, 197);
+  font-size: 12px;
+}
+
+#signin-button,
+#create-account {
+  border: none;
+  border-radius: 5px;
+  background-color: rgb(12, 97, 184);
+  color: rgb(255, 255, 255);
+  margin-right: 35px;
+  margin-top: -2px;
```

Comment:
## Use Shorthand Properties[1]

For instance, instead of using margin-top, margin-right, margin-bottom, margin-left separately you can simply use just one line.

```
#div {

margin: 10px 10px 3px 4px; // top, right, bottom left 

}
```

###### _[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_